cloud/google/clientcomputeservice_test.go

```diff
@@ -0,0 +1,96 @@
+package google_test
+
+import (
```

Comment:
if you only need a single import you can put it on one line instead of in a block

Reply:
Fixed.